src/main/java/uk/gov/hmcts/reform/roleassignment/oidc/JwtGrantedAuthoritiesConverter.java

```diff
@@ -51,7 +52,7 @@ private List<GrantedAuthority> extractAuthorityFromClaims(List<String> roles) {
         return roles.stream()
             .map(SimpleGrantedAuthority::new)
             .map(GrantedAuthority.class::cast)
-            .toList();
+            .collect(Collectors.toList());
```

Comment:
See [sonar report](https://sonarcloud.io/summary/new_code?id=am-role-assignment-service&pullRequest=1785)

Reply:
collect(Collectors.toList()) changed back to toList() . I think this was only change originally to fix a Sonarqube issue and now Sonarqube is complaining wanting it changed back!